tree/master/packages/extension/public/locales/pl/translation.json

```diff
@@ -0,0 +1,162 @@
+{
+	  "Copied": "Skopiowane",
+	  "<unknown>": "<nieznane>",
+	  "click to select or drag and drop the file here": "kliknij, aby wybrać lub przeciągnij i upuść plik tutaj",
+	  "{{name}} ({{size}} bytes)": "{{name}} ({{size}} bajtów)",
+	  "... loading ...": "... Ładowanie ...",
+	  "Generated 12-word mnemonic seed:": "Wygenerowane 12-wyrazowe ziarno mnemoniczne:",
```

Comment:
"Ziarno mnemoniczne" brzmi strasznie, ale widziałem, że pojawia się w innych narzędziach. Może "12-wyrazowe ziarno" wystarczy?

Reply:
Dzięki, jak uważasz. Nie jestem taki obeznany z polskim technicznym językiem a zwłaszcza jeżeli mówimy o blockchain. Tem chcialem to przetłumaczyć by sie trochę pod uczyć.

Reply:
Jasne, rozumiem :) To był jeden z pierwszych komentarzy, ale pojawia się też dość często w kolejnych liniach, więc zostawmy tak jak jest. Dzięki!!!